src/extra/python/scripts/remove_certain_restart_and_data_files.py

```diff
@@ -98,36 +125,18 @@ def keep_only_certain_daily_data_uninterp(exp_object, max_num_files, interval=No
             
 if __name__=="__main__":
 
-    max_num_files_input = 325
+    max_num_files_input = None
     
-#     exp_name_list=['simple_continents_post_princeton_qflux_anoms_'+str(x) for x in range(31,32)]
-
-#     exp_name_list=['aquaplanet_qflux_anoms_'+str(x) for x in [12,18,23,32,8]]
-
-#     exp_name_list = ['simple_continents_post_princeton_qflux_control_1','simple_continents_post_princeton_fixed_sst_1', 'simple_continents_post_princeton_qflux_control_nod_1', 'simple_continents_post_princeton_qflux_control_scf_1']
-#     
-#     exp_name_list = ['annual_mean_ice_princeton_qflux_control_matrix_qflux_2017_code_1', 'annual_mean_ice_post_princeton_fixed_sst_1', 'annual_mean_ice_princeton_fixed_sst_1'] 
-# 
-#     exp_name_list.extend(['annual_mean_ice_post_princeton_fixed_sst_el_nino_1'])
-
-#     exp_name_list = ['simple_continents_post_princeton_qflux_control_1']
-
-#     exp_name_list = ['annual_mean_ice_princeton_qflux_control_1']#, 'annual_mean_ice_post_princeton_qflux_control_1']
-
-#     exp_name_list = ['annual_mean_ice_post_princeton_fixed_sst_TEST_1', 'annual_mean_ice_princeton_qflux_control_matrix_qflux_1']
-
-#     exp_name_list.extend(['simple_continents_post_princeton_fixed_sst_1'])
-
-#     exp_name_list = ['giant_drag_exp_chai_values_without_dc_bug_latest_1']
-#     exp_name_list = ['aquaplanet_qflux_control_1']
+    # exp_name_list = ['']
+    exp_name_list = glob('/disca/share/sit204/data_from_isca_cpu/*/')
```

Comment:
could this be a non-hard coded option? If not, could you link at the top of the script so it is easy for a user to open and change.